### PR TITLE
Use keyword arguments

### PR DESCRIPTION
**Pull request by flora-hofmann-frequenz**
This will help fix nox error in PR #13.

### CODE REVIEW

**Review by cwasicki — DISMISSED** [review]

**Comment by cwasicki** [comment]
Oh, since we have already a release we need release notes :(

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -15,3 +15,4 @@
 ## Bug Fixes
 
 * Change 0.0 to nan if there is no data available
+* Make use of keyword arguments
```
Comment (cwasicki) [inline RELEASE_NOTES.md]: `Enforce keyword arguments in cumulative_energy function.`

**Review by cwasicki — APPROVED** [review]

**Comment by Marenz** [comment]
If you don't want to change your code accordingly (as in, use key-word arguments), you can also just add a new exception:
```
# pylint: disable-next=too-many-positional-arguments
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -15,3 +15,4 @@
 ## Bug Fixes
 
 * Change 0.0 to nan if there is no data available
+* Enforce keyword arguments in cumulative_energy function
```
Comment (llucax) [inline RELEASE_NOTES.md:18]: Nitpick, but this is not really a bug fix, I would say it's more of an improvement, and a breaking change, as now callers must use keyword arguments. So in the future you might want to mention it in the **Upgrading** section.

Comment (flora-hofmann-frequenz) [reply RELEASE_NOTES.md:18]: Will do so - we have the same topic coming up in the client, so I will change it there :-)

**Pull request merged**